docs/UserGuide.adoc

```diff
@@ -187,30 +187,30 @@ Format: `quit`
 ==== Get statistics for subjects : `stats`
 
 Returns a pie chart of the number of questions that have been answered correctly and incorrectly.
-You can choose to only get the statistics for a difficulty level by adding an optional `d/`.
-You can even get the combined statistics of more than one subject. +
-Format: `stats s/SUBJECT [d/DIFFICULTY] [s/SUBJECT1] [s/SUBJECT2]...`
+You can choose to get the statistics for a difficulty level by adding `d/` or
+get the statistics of different subjects by adding `s/`. +
+Format: `stats [d/DIFFICULTY] [s/SUBJECT1] [s/SUBJECT2]...`
 
 ==== Get questions with correct/incorrect answers: `questions`
-Gets all questions that have been answered correctly/incorrectly for a particular subject by using `-c/-i` respectively.
-Either one of `-c` or `-i` must be used.
-You can also choose to see past answers to the questions by adding an optional `-a`. +
-Format: `questions s/SUBJECT [-c] [-i] [-a]`
+Gets all questions that have been answered correctly/incorrectly by using `-c/-i` respectively.
+Either one of `-c` or `-i` must be used, you can also filter by subjects by adding `s/`. +
+Format: `questions [-c] [-i] [s/SUBJECT1] [s/SUBJECT2]...`
 
 ==== Get a report for individual questions: `report`
 Returns a report of how well you have answered a particular question,
 the number of times the question has been attempted and the past answers to the question. +
 Format: `report INDEX`
 
-==== Get overview of lecture notes: `overview -ln`
+==== Get overview of lecture notes: `overviewn`
 Gets an overview of the types of notes that have been added into the app.
 A stacked bar graph, sorted by subjects will be returned. +
-Format: `overview -ln`
+Format: `overviewn`
 
-==== Get overview of questions attempted: `overview -q`
+==== Get overview of questions attempted: `overviewq`
 Gets an overview of the types of questions that have been attempted overall.
-A stacked bar graph, sorted by subjects will be returned. +
-Format: `overview -q [dt/START_DATE] [dt/END_DATE]`
+A stacked bar graph, sorted by subjects will be returned.
+You can also add a `dt/` field to get questions that you have done within that time period. +
+Format: `overviewq [dt/START_DATE] [dt/END_DATE]`
```

Comment:
You haven't added these commands to the application yet, no?

Reply:
I have added overviewq without the dt not too sure about overviewn, I am thinking of removing it since there's no need to sort notes subjects.

Reply:
Yeah, remove `overviewn`.

Reply:
Okay